app/src/main/res/values-fr/help.xml

```diff
@@ -0,0 +1,247 @@
+        à la recherche des transactions appartenant à votre adresse. Cela prend <em>beaucoup</em> de temps.</p>
+    ]]></string>
+
+    <string name="help_create_keys"><![CDATA[
+        <h1>Créer un Portefeuille - Clefs</h1>
+        <p>Si vous récupérez votr portefeuille en utilisant vos clefs !</p>
+        <p>Entrez un nom et un mot de passe unique pour le portefeuille.
+        Le mot de passe permet de sécuriser les données de votre portefeuille sur votre appareil.
+        Utilisez un mot de passe fort - mieux, utilisez une phrase secrète.<p>
+        <p>Entrez votre adresse Monero dans le champ \"Adresse Publique\" et remplissez \"Clef d’Audit\" et \"Clef de dépense\".</p>
+        <p>Si vous connaissez le numéro du bloc de votre première transaction à cette adresse, entrez-le dans le
+        champ \"Hauteur de restauration\" - en le laissant vide la <em>totalité</em> de la chaîne de blocs sera scannée
+        à la recherche des transactions appartenant à votre adresse. Cela prend <em>beaucoup</em> de temps.</p>
+    ]]></string>
+
+    <string name="help_create_view"><![CDATA[
+        <h1>Créer un Portefeuille - Audit</h1>
+        <p>Si vous voulez suveiller les transaction entrantes sur un portefeuille !</p>
+        <p>Entrez un nom et un mot de passe unique pour le portefeuille.
+        Le mot de passe permet de sécuriser les données de votre portefeuille sur votre appareil.
+        Utilisez un mot de passe fort - mieux, utilisez une phrase secrète.<p>
+        <p>Entrez votre adresse Monero dans le champ \"Adresse Publique\" et remplissez la \"Clef d’Audit\".</p>
+        <p>Si vous connaissez le numéro du bloc de votre première transaction à cette adresse, entrez-le dans le
+        champ \"Hauteur de restauration\" - en le laissant vide la <em>totalité</em> de la chaîne de blocs sera scannée
+        à la recherche des transactions appartenant à votre adresse. Cela prend <em>beaucoup</em> de temps.</p>
+    ]]></string>
+
+    <string name="help_details"><![CDATA[
+        <h1>Détails du Portefeuille</h1>
+        <h2>Adresse Publique</h2>
+        Votre adresse publique est comme votre numéro de compte en banque, vous pouvez la partager avec n’importe qui sans
+        craindre de perdre vos Moneroj. On porra envoyer des Moneroj dans votre portefeuille en utilisant cette adresse.
+        <h2>Phrase Mnémonique</h2>
+        C’est la seule donée nécessaire pour récupérer votre portefeuille ultérieurement et obtenir l’accès à
+        vos fonds. Il est très imprtant de la garder sécurisée et confidentielle, car elle donne <em>à quiconque</em>
+        le contrôle total de vos Moneroj ! Si vous ne l’avez pas écrite à un emplacement sécurisé, faites-le !
+        <h2>Mdp de restauration des fichiers du Portefeuille</h2>
+        Assurez vous de noter ce mot de passe. Si vous réinitialisez votre appareil ou désinstallez l'application,
```

Comment:
pls excape the ```'``` - why is the apostrophe a different one than e.g. "n’importe" or "l’accès"?